cms/envs/devstack_docker.py

```diff
@@ -9,7 +9,8 @@
 
 LOGGING['loggers']['tracking']['handlers'] = ['console']
 
-LMS_ROOT_URL = 'http://edx.devstack.lms:18000'
+LMS_BASE = 'edx.devstack.lms:18000'
```

Comment:
Is CMS_BASE not needed here, but only for CMS?  I'm not clear on why it was added below with no references to it, but I imagine the references already exist?